### PR TITLE
Reinstate some content removed by #10

### PR DESCRIPTION
**Pull request by eskatos**
#10 simplified the document's first sections
This PR reinstate some key information that was lost.

### CODE REVIEW

File: contents/index.adoc [hunk 1]
```diff
@@ -24,11 +24,15 @@ image::intellij-idea-android-studio.png[IntelliJ IDEA and Android Studio]
 **Please read:** It's helpful to understand the following important information _before you migrate_:
 
 * Some IDEs, such as Eclipse or NetBeans, do not yet provide helpful tools for editing Gradle Kotlin DSL files, however, importing and working with Kotlin DSL-based builds work as usual.
-* In IntelliJ, you must link:https://www.jetbrains.com/help/idea/gradle.html#gradle_import[import your project from the Gradle model] to get content-assist and refactoring tools for Kotlin DSL scripts.
+* In IntelliJ IDEA, you must link:https://www.jetbrains.com/help/idea/gradle.html#gradle_import[import your project from the Gradle model] to get content-assist and refactoring tools for Kotlin DSL scripts.
+* IntelliJ IDEA or Android Studio might spawn up to 3 Gradle daemons when editing scripts. One per each type of scripts: project scripts, settings scripts and initialization scripts. Also note that there are reports that on some builds with slow configuration time the IDE may feel unresponsive when editing scripts.
```
Comment (eriwen) [inline contents/index.adoc]: Could we consider just 1 point here about performance instead of 3? I don't think we need all of the detail we have, because detailed statements will likely become incorrect.

How about the following to replace lines 28, 34, and 35:

> The performance characteristics of the Groovy and Kotlin DSLs are generally similar, but certain scenarios, like the first build in a session, are known to be noticeably slower. We're working hard to make this better. 

When encountering slowness, almost no one is going to have the number of daemons or what part of their build they're changing on their minds. They just care that sometimes IDE sync feels a lot slower than usual.

File: contents/index.adoc [hunk 2]
```diff
@@ -24,11 +24,15 @@ image::intellij-idea-android-studio.png[IntelliJ IDEA and Android Studio]
 **Please read:** It's helpful to understand the following important information _before you migrate_:
 
 * Some IDEs, such as Eclipse or NetBeans, do not yet provide helpful tools for editing Gradle Kotlin DSL files, however, importing and working with Kotlin DSL-based builds work as usual.
```
Comment (eriwen) [inline contents/index.adoc]: Perhaps we should add a point saying: "You should be on the latest version of Gradle and IDEs to have the best experience"?

Comment (eskatos) [reply contents/index.adoc]: Done

File: contents/index.adoc (hunk 1 above)
Comment (eskatos) [reply contents/index.adoc]: These statements are correct Today. Of course they will eventually become incorrect and this guide is part of what can/will be refactored.

Once again I think we should not underestimate the audience and that we should openly state the current issues.

The number of daemons is crucial for memory hungry builds such as Android ones. First-use and "change in buildSrc" are important use cases and we don't want to put the "3x-4x slower" situation under the carpet.

Now, maybe current words sounds too much like a "laundry list", please suggest rewording.

File: contents/index.adoc [hunk 3]
```diff
@@ -23,9 +23,15 @@ image::intellij-idea-android-studio.png[IntelliJ IDEA and Android Studio]
...
+The same applies to the scenario in which something in the _buildSrc_ directory changes, which invalidates build-script caching.
+In addition, IntelliJ IDEA or Android Studio might spawn up to 3 Gradle daemons when editing scripts; one per each type of script: project scripts, settings scripts and initialization scripts.
+On some builds with slow configuration time (please check out the link:https://guides.gradle.org/performance/#configuration[performance guide]) the IDE may feel unresponsive when editing scripts.
 * You must run Gradle with Java 8 or higher. Java 7 is not supported.
+* The embedded Kotlin compiler is known to work on Linux, macOS, Windows, Cygwin, FreeBSD and Solaris on x86-64 architectures.
 * Knowledge of Kotlin syntax and basic language features is very helpful. The link:{kotlin-reference}[Kotlin reference documentation] and link:https://kotlinlang.org/docs/tutorials/koans.html[Kotlin Koans] should be useful to you.
 * Use of the `plugins {}` block to declare Gradle plugins significantly improves the editing experience, and is highly recommended. Consider adopting it in your Groovy build scripts before converting them Kotlin.
```
Comment (thc202) [inline contents/index.adoc]: Hopefully it's fine to add the comment in this pull request, `... converting them *to* Kotlin`?

Comment (eskatos) [reply contents/index.adoc]: Thank you @thc202, fixed!

Comment (thc202) [reply contents/index.adoc]: Thanks!
